packages/components/src/utils/theme-variables.scss

```diff
@@ -2,10 +2,10 @@
 //
 // If the `--wp-components-color-accent` variable is not defined, fallback to
 // `--wp-admin-theme-color`. If the `--wp-admin-theme-color` variable is not
-// defined, fallback to the default theme color (WP blue).
-$components-color-accent: var(--wp-components-color-accent, var(--wp-admin-theme-color, #007cba));
-$components-color-accent-darker-10: var(--wp-components-color-accent-darker-10, var(--wp-admin-theme-color-darker-10, #006ba1));
-$components-color-accent-darker-20: var(--wp-components-color-accent-darker-20, var(--wp-admin-theme-color-darker-20, #005a87));
+// defined, fallback to the default theme color (WP blueberry).
+$components-color-accent: var(--wp-components-color-accent, var(--wp-admin-theme-color, #3858e9));
+$components-color-accent-darker-10: var(--wp-components-color-accent-darker-10, var(--wp-admin-theme-color-darker-10, #2145e6));
+$components-color-accent-darker-20: var(--wp-components-color-accent-darker-20, var(--wp-admin-theme-color-darker-20, #183ad6));
```

Comment:
I know these accents exist already, do you know where they are used?

Also, the wp.org site has a larger range of blues, so despite my previous comment, perhaps we should use [those](https://www.figma.com/file/HOJTpCFfa3tR0EccUlu0CM/WordPress.org-Design-Library?type=design&node-id=101%3A4031&t=vox6KwIW9vMeBq7V-1):

* Blueberry: #3858E9
* Deep Blueberry (or darker-10): #213FD4
* Dark Blueberry (or darker-20): #1d35b4

Reply:
Here are [all the usages](https://github.com/search?q=repo%3AWordPress%2Fgutenberg+%28%24components-color-accent-darker+OR+themeDark10+OR+--wp-admin-theme-color-darker%29+NOT+path%3A%2F%5Epackages%5C%2Fcomponents%5C%2Fsrc%5C%2Futils%5C%2F%2F+NOT+path%3A%2F__snapshots__%2F+NOT+path%3A%2F%5Epackages%5C%2Fbase-styles%2F&type=code) of the darker accent colors in components (all packages) right now.